### DISCUSSION TITLE
Why a perfectly valid accessibility issue was closed with zero response

### DISCUSSION THREAD

**Discussion by steverep** [renovatebot/renovate#20789] [Suggest an Idea]
### Type of discussion.

I just want to chat

### Tell us more.

Since it was rudely marked as invalid, closed, and locked on me by @rarkins, I guess I need to ask here why on Earth #20783 was deemed invalid?  The inaccessible markdown is easily verified on any Renovate PR that includes the images, so a minimal reproduction or log is not relevant at all.

**Comment by rarkins** [comment]
First you ignored these directions:
<img width="633" alt="image" src="https://user-images.githubusercontent.com/6311784/223337063-eba4d13f-3a7b-487c-a525-33d453fc0b89.png">

Then you ignored these directions:
<img width="914" alt="image" src="https://user-images.githubusercontent.com/6311784/223337164-0cbf82e1-9e85-41b1-b66c-8db17cc6a1c1.png">

Now you're wasting further time of yourself and me by complaining. The markdown you refer to isn't even in this repo, but you're welcome to start another Discussion about it. I consider this Discussion here answered and resolved
